gn3/settings.py

```diff
@@ -57,3 +57,5 @@ def parse_env_cors(default):
 TEXTDIR = f"{GNSHARE}/web/ProbeSetFreeze_DataMatrix"
 
 ROUND_TO = 10
+
+MULTIPROCESSOR_PROCS = 6 # Number of processes to spawn
```

Comment:
**nitpick:**  @zsloan and @fredmanglis we should make the default for this and make the value that works well on prod the default ;)

Reply:
Yes indeed! @zsloan You can experiment and change the value, I guess...